src/Microsoft.Extensions.Configuration.Json/JsonConfigurationProvider.cs

```diff
@@ -52,18 +52,20 @@ public override void Load(Stream stream)
 
         private static string RetrieveErrorContext(JsonReaderException e, IEnumerable<string> fileContent)
         {
-            string errorLine;
+            string errorLine = null;
             if (e.LineNumber >= 2)
             {
                 var errorContext = fileContent.Skip(e.LineNumber - 2).Take(2).ToList();
-                errorLine = errorContext[0].Trim() + Environment.NewLine + errorContext[1].Trim();
+                if (errorContext.Count() >= 2)
```

Comment:
When I finally understood what is going on my first three thought were:

1. The reported line number for the error y way off, possibly a bug in Json.NET that we need to compensate for.
2. We could ensure that the line we start looking for text at is not greater than `fileContent.Count() - 2`
3. It seems in this case the las string in `fileContent` is null. Perhaps that signals the end of file just after a line feed? Maybe we should special case a null line to be printed as `eof`? last line of the file content